scripts/lib/CIME/build.py

```diff
@@ -9,6 +9,29 @@
 
 logger = logging.getLogger(__name__)
 
+def get_standard_makefile_args(case):
+    variables = ["CASEROOT", "CASETOOLS", "CIMEROOT", "COMP_INTERFACE",
+                 "COMPILER", "DEBUG", "EXEROOT", "INCROOT", "LIBROOT",
+                 "MACH", "MPILIB", "NINST_VALUE", "OS", "PIO_VERSION",
+                 "SHAREDLIBROOT", "SMP_PRESENT", "USE_ESMF_LIB", "USE_MOAB",
```

Comment:
Need USE_KOKKOS here, at least for e3sm.

Reply:
I put it in line 497 of this file, still as an environment variable - to be consistent it should be a case variable as are USE_TRILINOS and USE_ALBANY at line 467+